### ISSUE TITLE
Use dynamic page titles

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/remote-office-hours-queue#436]
@janelilr from the DUX team gave us the following feedback:

"The one thing I noticed, is the Tab at the top of the browser page has a title, but it doesn't change to reflect the page you are on. Example: Remote Offices Hours Queue - add queue."

[WCAG 2.1.4](https://www.w3.org/WAI/WCAG21/Understanding/page-titled) mentions this in its examples and has it flagged as an Advisory Technique. I believe we've done this in the past using [`react-helmet`](https://www.npmjs.com/package/react-helmet).

**Comment by Nyu10** [comment]
Hi, me and @catzhaang will take this

**Comment by jonespm** [comment]
React-Helmet-Async is the current solution and was partially implemented in another issue.

https://www.npmjs.com/package/react-helmet-async

**Comment by PraveshKunwar** [comment]
this has been fixed in the linked pr

**Comment by jonespm** [comment]
Hi @PraveshKunwar, it looks like the PR is still  open. When a linked PR is merged and it will automatically close this issue. You don't need to manually close it early.

**Comment by HemanginiShah** [comment]
I am not seeing the info on the tab.  I may need to do slack huddle with Pravesh, why I am not seeing the info when I hover over Tab on the screen.
![Image](https://github.com/user-attachments/assets/8bda3d89-de09-46dc-b03e-89190be6a396)

**Comment by HemanginiShah** [comment]
I tried to check the top of the tab to confirm it displays /page/, /manage/, or /queue/ as the user navigates through different sections of the app.  I didn't observe any changes.

I may need to huddle with Pravesh to discuss this issue.
